types/better-sqlite3/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for better-sqlite3 3.1
+// Project: http://github.com/JoshuaWise/better-sqlite3
+// Definitions by: Ben Davies <https://github.com/Morfent/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as Integer from 'integer';
+
+interface RunResult {
+	changes: number;
+	lastInsertRowID: number;
```

Comment:
this is misspelled, it should be `lastInsertROWID`, and it can be a `number` or a `Database.Integer`